app/javascript/src/actions/utils/crud.js

```diff
@@ -1,36 +1,45 @@
-import axios from 'axios';
-import { push } from 'react-router-redux';
+import axios from "axios";
+import { push } from "react-router-redux";
 
-const crud = request => (dispatch) => {
+//const { INTERNAL_API_KEY } = process.env;
+const INTERNAL_API_KEY = "hellopass";
+
+const crud = request => dispatch => {
   dispatch({
-      type: request.dispatch.begin,
-    });
+    type: request.dispatch.begin
+  });
+
+  const secureRequestData = {
```

Comment:
This will not work for a GET request as they do not take data element. Did you test GET requests with a `api_key` query added to the url like: `..../api/v1/user?test=blah&api_key=blahblah`?